tests/app/notifications/test_validators.py

```diff
@@ -0,0 +1,160 @@
+    assert service_can_send_to_recipient("some_other_email@test.com",
+                                         key_type,
+                                         live_service) is None
+    assert service_can_send_to_recipient('07513332413',
+                                         key_type,
+                                         live_service) is None
+
+
+@pytest.mark.parametrize('key_type',
+                         ['team'])
+def test_service_can_send_to_recipient_passes_for_whitelisted_recipient_passes(key_type, notify_db, notify_db_session,
+                                                                               sample_service):
+    sample_service_whitelist(notify_db, notify_db_session, email_address="some_other_email@test.com")
+    assert service_can_send_to_recipient("some_other_email@test.com",
+                                         key_type,
+                                         sample_service) is None
+    sample_service_whitelist(notify_db, notify_db_session, mobile_number='07513332413')
+    assert service_can_send_to_recipient('07513332413',
+                                         key_type,
+                                         sample_service) is None
+
+
+@pytest.mark.parametrize('recipient', ['07513332413', 'some_other_email@test.com'])
+@pytest.mark.parametrize('key_type, error_message',
+                         [('team', 'Can’t send to this recipient using a team-only API key'),
+                          ('normal',
+                           "Can’t send to this recipient when service is in trial mode – see https://www.notifications.service.gov.uk/trial-mode")])  # noqa
+def test_service_can_send_to_recipient_fails_when_recipient_is_not_on_team(recipient, key_type, error_message,
+                                                                           notify_db, notify_db_session):
+    trial_mode_service = create_service(notify_db, notify_db_session, service_name='trial mode', restricted=True)
+    with pytest.raises(BadRequestError) as exec_info:
+        assert service_can_send_to_recipient(recipient,
+                                             key_type,
+                                             trial_mode_service) is None
+    assert exec_info.value.status_code == 400
+    assert exec_info.value.code == 10400
+    assert exec_info.value.message == error_message
+    assert exec_info.value.link == 'link to documentation'
+    assert exec_info.value.fields == []
+
+
+def test_service_can_send_to_recipient_fails_when_mobile_number_is_not_on_team(notify_db, notify_db_session):
+    live_service = create_service(notify_db, notify_db_session, service_name='live mode', restricted=False)
+    with pytest.raises(BadRequestError) as e:
+        assert service_can_send_to_recipient("0758964221",
+                                             'team',
+                                             live_service) is None
```

Comment:
since you expect it to raise, you don't need this `assert is None` bit (as if it doesn't raise and does return, that'll fail the test anyway)